sweddit_combined/decisions/ChineseFormables.txt

```diff
@@ -154,19 +87,46 @@ country_decisions = {
 				}
 				add_permanent_claim = IMP
 			}
-			add_prestige = 25
+			add_prestige = 50
 			set_government_rank = 3
 			if = {
 				limit = { has_custom_ideas = no }
 				country_event = { id = ideagroups.1 } #Swap Ideas
 			}
+			if = {
+				limit = { religion = confucianism }
+				add_government_reform = mandate_of_heaven_reform
+				add_government_reform = imperial_examination_reform
+			}
 			set_country_flag = formed_chinese_empire
-			add_government_reform = chinese_empire
-			add_government_reform = imperial_examination
+			add_government_reform = chinese_empire_reform
 		}
 		ai_will_do = {
 			factor = 1
 		}
 		ai_importance = 400
 	}
+
+	adopt_confucian_administration = {
```

Comment:
Should normal conversion to confucian_state_reform really exist if this decision exists?